packages/ag-charts-community/src/scene/transformable.ts

```diff
@@ -131,16 +131,18 @@ function MatrixTransform<N extends Node>(Parent: Constructor<N>) {
             const matrix = this[TRANSFORM_MATRIX];
 
             let performRestore = false;
-            if (!matrix.identity) {
-                ctx.save();
-                performRestore = true;
-                matrix.toContext(ctx);
-            }
-
-            super.render(renderCtx);
-
-            if (performRestore) {
-                ctx.restore();
+            try {
```

Comment:
Again this will have a performance impact

Reply:
Not visible in `/benchmarks` - I agree that in the event of errors being thrown there is a performance hit, but for the normal non-error case there should be negligible overhead for this construct.